platemapper/main.py

```diff
@@ -1,65 +1,36 @@
 from platemapper import filehandler
-import ordinationbuild
-import qiimebuild
+from platemapper import ordinationbuild
+from platemapper import qiimebuild
 import pandas as pd
-# -----------------------------
-import tkinter
-from tkinter import filedialog
-# -----------------------------
+
 import shutil
 #  import os
 # execute everything
 
 # make folderstructure
 
-filehandler.makefolder()
-
+import qiime2
+import qiime2.plugins.emperor.actions as emperor_actions
 
-def load_file():
-    """
-    Selecting a metafile (e.g. meta_plate.tsv). Right now
-    this happens via tkinter, but this will be erased soon
-    as quita doesnt need this. No idea as how it will have
-    to be implemented
-
-    Raises:
-        KeyError:   If no File Path exists, there will be no
-                    file chosen by user. User has to choose
-                    a File.
-
-    Returns:
-        df (pd.DataFrame):  metafile as pd.DataFrame for
-                            further manipulation
-
-        file_path (str; path):  filepath for copying the metafile
-                                into the correct position and
-                                for extracting the Foldername.
-    """
-    window = tkinter.Tk()
-    window.minsize(1000, 1000)
-    window.withdraw()    # tkinter.Tk().withdraw()
-    file_path = filedialog.askopenfilename(parent=window,
-                                           filetypes=[("TSV files", "*.tsv")],
-                                           title="choose file:")  # choose file
```

Comment:
the Qiita mechanics work such that you can expect to get a DataFrame passed (we will work on the "prep" (not the "sample") metadata